tailwind.config.js

```diff
@@ -38,10 +38,13 @@ module.exports = {
       },
       flex: {
         '1-0-0': '1 0 0%',
+      },
+      screens: {
+        'sm': {
+          max: '430px'
+        }
       }
     },
   },
-  plugins: [
-    require('@tailwindcss/line-clamp'),
-  ],
+  plugins: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've removed the '@tailwindcss/line-clamp' plugin from the list of plugins. This could potentially break any part of your application that relies on this plugin for text truncation. Please make sure to check all instances where you have used this plugin and verify if they still work as expected after this removal.
```diff
-  plugins: [],
+  plugins: [
+    require('@tailwindcss/line-clamp'),
+  ],
```
If you intentionally want to remove this plugin, ignore the above suggestion.


<!-- This is an auto-generated comment by OSS CodeRabbit -->